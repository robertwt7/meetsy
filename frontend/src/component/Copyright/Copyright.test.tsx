import { render } from "src/testutils/test-utils";
import { screen } from "@testing-library/react";
import { Copyright } from "./Copyright";

jest.mock("next/router", () => ({
  useRouter: jest.fn().mockReturnValue({
    pathname: "/",
  }),
}));

describe("Copyright Component", () => {
  test("should match snapshot", () => {
    const { baseElement } = render(<Copyright />);

    expect(baseElement).toMatchSnapshot();
  });

  test("should render Meetsy word", () => {
    render(<Copyright />);

    expect(screen.getByText("Meetsy")).toBeTruthy();
  });
});
