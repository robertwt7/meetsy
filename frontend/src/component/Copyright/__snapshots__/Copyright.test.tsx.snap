// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright Component should match snapshot 1`] = `
<body>
  <div>
    <p
      class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
    >
      Copyright Â© 
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1kgfegd-MuiTypography-root-MuiLink-root"
        href="http://localhost:3000"
      >
        Meetsy
      </a>
       
      2022
      .
    </p>
  </div>
</body>
`;
